PlanGrid.Api/Attachment.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <copyright file="Attachment.cs" company="PlanGrid, Inc.">
+//     Copyright (c) 2015 PlanGrid, Inc. All rights reserved.
+// </copyright>
+
+using System;
+using Newtonsoft.Json;
+
+namespace PlanGrid.Api
+{
+    public class Attachment
+    {
+        [JsonProperty("uid")]
```

Comment:
Because it introduces fewer dependencies.  `JsonProperty` is part of JSON.NET, whereas `DataMember` requires using `System.Runtime.Serialization`.  Probably not a big deal, but it seems since we're using Json.Net, it makes sense to use the attributes provided by the framework.